loculus_values/values.yaml

```diff
@@ -2509,7 +2509,7 @@ organisms:
           nextclade_dataset_name: "nextstrain/rsv/a/EPI_ISL_412866"
           nextclade_dataset_tag: "2024-11-27--02-51-00Z"
           require_nextclade_sort_match: true
-          minimizer_url: "https://anna-parker.github.io/InfluenzaAReferenceDB/results/rsv_segments.minimizer.json"
+          minimizer_url: "https://github.com/alejandra-gonzalezsanchez/loculus-evs/blob/master/rsv_minimizer-index.json"
```

Comment:
before merging we should host on github pages or similar

Reply:
I have just created a new repo on Loculus that is hosted on github pages